packages/cx/src/widgets/grid/Grid.js

```diff
@@ -1303,6 +1306,13 @@ class GridComponent extends VDOM.Component {
             mod["non-selectable"] = !selectable;
          }
 
+         let draggable =
+            dragSource &&
+            (!row.dragHandles || row.dragHandles.length == 0) &&
+            (!isRecordDraggable || isRecordDraggable(record.data));
+         mod["draggable"] = draggable;
+         mod["non-draggable"] = !draggable;
+
```

Comment:
I think this part can be removed as draggability does not influence the visual appearance of the row in any way, but I might be wrong.

Reply:
`draggable` mod was already there, I've just added `non-draggable` to make it equivalent to the selection mods. My idea was that non-draggable records could be visually differentiated from the the ones that are draggable, but not sure whether it's a valid use case.